detections/endpoint/macos_amos_stealer___virtual_machine_check_activity.yml

```diff
@@ -0,0 +1,78 @@
+data_source:
+  - osquery
+search: |
+  `osquery_macro` name=es_process_events 
+  columns.cmdline="*osascript*" AND columns.cmdline="* -e *" AND columns.cmdline="*set*" AND columns.cmdline="*system_profiler*" AND columns.cmdline IN ("*VMware*", "*QEMU*") 
+  | rename columns.* as * 
+  | stats  min(_time) as firstTime max(_time) as lastTime 
+    values(cmdline) as cmdline, 
+    values(pid) as pid, 
+    values(parent) as parent, 
+    values(path) as path,
+    values(signing_id) as signing_id,  
+    by username host 
+  | rename 
+    username as user, 
+    cmdline as process, 
+    parent as parent_process, 
+    path as process_path, 
+    host as dest 
+  | `security_content_ctime(firstTime)` 
+  | `security_content_ctime(lastTime)`
+  | `macos_amos_stealer___virtual_machine_check_activity_filter`
+how_to_implement: |
+  This detection leverages osquery and endpoint security on MacOS. Follow the link in references, which describes how to setup process auditing in MacOS with endpoint security and osquery.
+known_false_positives: None identified.
+references:
+  - https://osquery.readthedocs.io/en/stable/deployment/process-auditing/
+  - https://www.virustotal.com/gui/search/behaviour_processes%253A%2522osascript%2520-e%2520set%2522%2520AND%2520behaviour_processes%253A%2522system_profiler%2522%2520AND%2520(behaviour_processes%253A%2522VMware%2522%2520OR%2520behaviour_processes%253A%2522QEMU%2522)?type=files
+drilldown_searches:
+  - name: View the detection results for - "$user$" and "$dest$"
+    search: '%original_detection_search% | search  user = "$user$" dest = "$dest$"'
+    earliest_offset: $info_min_time$
+    latest_offset: $info_max_time$
+  - name: View risk events for the last 7 days for - "$user$" and "$dest$"
+    search: '| from datamodel Risk.All_Risk | search normalized_risk_object IN ("$user$",
+      "$dest$") starthoursago=168  | stats count min(_time) as firstTime max(_time)
+      as lastTime values(search_name) as "Search Name" values(risk_message) as "Risk
+      Message" values(analyticstories) as "Analytic Stories" values(annotations._all)
+      as "Annotations" values(annotations.mitre_attack.mitre_tactic) as "ATT&CK Tactics"
+      by normalized_risk_object | `security_content_ctime(firstTime)` | `security_content_ctime(lastTime)`'
+    earliest_offset: $info_min_time$
+    latest_offset: $info_max_time$
+rba:
+  message: AMOS Stealer activity on host $dest$ by user $user$
+  risk_objects:
+    - field: user
+      type: user
+      score: 40
+    - field: dest
+      type: system
+      score: 40
+  threat_objects: []
+tags:
+  analytic_story: []
```

Comment:
@nasbench - we would need to tag this to an analytic story- Lets consider create a new story file for : MacOS AMOS Stealer ?